projects/netgrif-components-core/src/lib/view/task-view/abstract-single-task-view.component.ts

```diff
@@ -18,22 +17,25 @@ export abstract class AbstractSingleTaskViewComponent extends AbstractViewWithHe
 
     @Input() initiallyExpanded: boolean = true;
     @Input() preventCollapse: boolean = true;
-    public task$: Observable<TaskPanelData>;
+    public taskPanelData: BehaviorSubject<TaskPanelData>;
     public loading$: Observable<boolean>;
     private transitionId: string;
     private subRoute: Subscription;
-    protected unsubscribe$: Subject<void>;
+    protected subPanelData: Subscription;
 
     protected constructor(protected taskViewService: TaskViewService,
                           activatedRoute: ActivatedRoute) {
         super(taskViewService, activatedRoute);
-        this.unsubscribe$ = new Subject<void>();
+        this.subPanelData = new Subscription();
+        this.taskPanelData = new BehaviorSubject<TaskPanelData>(undefined);
         this.subRoute = this._activatedRoute.paramMap.subscribe(paramMap => {
             if (!!(paramMap?.['params']?.[TaskConst.TRANSITION_ID])) {
                 this.transitionId = paramMap['params'][TaskConst.TRANSITION_ID];
-                this.task$ = this.taskViewService.tasks$.pipe(map<Array<TaskPanelData>, TaskPanelData>(tasks => {
-                    return this.resolveTransitionTask(tasks);
-                }), takeUntil(this.unsubscribe$));
+                this.subPanelData = this.taskViewService.tasks$.subscribe(tasks =>  {
```

Comment:
if this fires multiple times, the first subscription reference is lost, but the subscription itself still exists.

If `subPanelData` is defined `.unsubscribe()` should be called.